<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeForm</class>
 <widget class="QWidget" name="UpgradeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBtn">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Installation detected.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abortBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>540</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="upgradeBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>108</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="newBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>188</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New Installation</string>
   </property>
  </widget>
  <widget class="QLabel" name="newLabel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>220</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new installation and remove all existing node configuration. Node will need to be re-deployed. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="upgradeLabel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>140</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Abort installation now. If the intent is to perform upgrade, please refer to the online upgrade guide. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select required action:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>upgradeBtn</tabstop>
  <tabstop>newBtn</tabstop>
  <tabstop>nextBtn</tabstop>
  <tabstop>prevBtn</tabstop>
  <tabstop>abortBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
